cluster/get-kube.sh

```diff
@@ -63,7 +63,49 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+KUBERNETES_RELEASE_URL_OVERRIDDEN=false
+KUBERNETES_CI_RELEASE_URL_OVERRIDDEN=false
+
+if [[ -v KUBERNETES_RELEASE_URL ]]; then
+    KUBERNETES_RELEASE_URL_OVERRIDDEN=true
+fi
+if [[ -v KUBERNETES_CI_RELEASE_URL ]]; then
+    KUBERNETES_CI_RELEASE_URL_OVERRIDDEN=true
+fi
+
 KUBERNETES_RELEASE_URL="${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release/release}"
```

Comment:
default for `KUBERNETES_RELEASE_URL` can now be `https://dl.k8s.io`.
default for `KUBERNETES_CI_RELEASE_URL` can now be `https://dl.k8s.io/ci`.

appending the semantic version to either (i.e. the vX.Y.Z+nnn) should do the right thing.